SU-SWS/decanter
# Includevar

# READY FOR REVIEW

# Summary
- Change extends paths to template variables that can be overridden 
- This change is to make it easier for other groups to change the path to the nested templates. 
- For example, I want to change the path to the card template so that it uses a different namespace.

# Needed By (Date)
- ?

# Urgency
- ?

# Steps to Test

1. Check out this branch and review the code
2. Build the styleguide
3. Ensure nothing changes.

# Affected Projects or Products
- Integrations


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
